modules/lib/options.nix

```
@@ -0,0 +1,12 @@
+{lib}: let
+  inherit (lib.options) mkOption;
+  inherit (lib.types) bool;
+in {
+  mkIntegrationOption = from: to:
+    mkOption {
+      default = false;
+      example = true;
+      description = "Whether to enable ${from} integration with ${to}.";
+      type = bool;
+    };
```

```suggestion
    mkEnableOption "${from} integration with ${to}";
```

Now that I'm looking at it like this I'm not so sure it's worth it, unless this makes it easier to write the integrations into `config`.

---

It does not make the integrations easier, what it does is just allow us to have a common, more semantically defined interface for them. So if we ever want to change the description, or anything about these options we can do it in one place